README.md

```diff
@@ -2,62 +2,160 @@
+and monitors (no mouse or graphic interface!) that were tied to a computer that
+they all shared. This monitor + keyboard device was called a "terminal." It looked
+like this:
+
+< pic of terminal >
+
+< graphic of terminal cabled to a shared computer >
+
+Nowadays, the "terminal" is "emulated" in software. It's "virtual." You launch
+the "emulator" by opening a program called `Terminal` on OSX or something
+similar on Linux or Windows. Instead of being connected to a remote computer by
+a cable, your "terminal emulator" talks to the computer you're actually typing
+on.
+
+< graphic of terminal cabled to itself >
+
+When you launch the "terminal emulator" program, it will immediately start a
+program called a _shell_ program. The _shell_ program is what actually prompts
+you for input and returns the output. The shell most computers default to
+using is known as `bash`.
+
+To help keep these terms straight, here's a table
+
+< insert table derived from style guide >
+
+## Demonstrate How to Navigate With `bash`
+
+To sum up: `bash` is a text-based interpreter that provides a _command-line
+interface_ for controlling your computer (or operating system).
+
+> **ASIDE**: Bash is actually an acronym which stands for **B**ourne-**A**gain
+> **SH**ell. As the word "again" suggests, there are _other_ shells, some of
+> which came before `bash`. There are also shells that have come along _since_
+> `bash`. Nevertheless most programmers use `bash` or something very similar.
+
+### Identify My Logged-In Username
+
+Let's start simply. Let's ask the computer who I am logged in as:
+
+```bash
+$ whoami
+```
+
+ < probably need screen shot>
+
+The `whoami` command lets you see which user account you're logged in to from a
+terminal window. This might seem obvious, especially if you're logged in on your
+personal computer, but it's not always clear which account you're you're working
+in the command line.
+
+In a strange circumstance where `whoami` isn't installed, there is another command
+you can use that can tell you your current username. Try:
+
+```bash
+$ id -un
+```
 
 ### Identifying the "Present Working Directory" With `pwd`
 
-Open up command prompt or terminal. Type in: `pwd` and hit return.
+With your Terminal program open, type in `pwd` and hit return/enter.
 
 ```bash
 $ pwd
 ```
 
 You should see some output describing the directory you are currently within.
+The `pwd` command stands for "**p**rint **w**orking **d**irectory".
+You'll see that the operating system created a directory under the "User"
+directory that was named the same as your `whoami` result
 
-`/Users/[your user name]]`
+`/Users/[your user name]`
 
-That output is describing a location on your computer. You have a file system and within that
-file system are directories and files.
+We would call this directory your "home" directory.
 
-The `pwd` command stands for "**p**rint **w**orking **d**irectory".
+That output is describing a location on your computer.
```

Comment:
Add:

<insert screenshots showing navigating from / to Users to kellyegreene with Finder>
<insert screenshot of pwd>

Comment:

We understand that it's describing a location, but we need to show the nesting folders kinda GUI experience for comparison. Probably also a pic of a file tree something like:

<pre>
/
  etc
  var
  tmp
  Users
    kellyegreene
    myannoyingbrother
    grandma
</pre>